src/hivpy/experiment.py

```diff
@@ -1,34 +1,59 @@
 from dataclasses import dataclass
-
-from .population import Population
+from datetime import date, timedelta
 from .simulation import run_simulation, SimulationConfig
+import logging
+import os
 
+LEVELS = {
+    'DEBUG': logging.DEBUG,
+    'INFO': logging.INFO,
+    'WARNING': logging.WARNING,
+    'ERROR': logging.ERROR,
+    'CRITICAL': logging.CRITICAL
+}
 
 @dataclass
-class ExperimentConfig:
-    """Configuration parameters covering all stages of an experiment."""
-    population_size: int
-    simulation: SimulationConfig
+class OutputConfig:
+    output_dir: str
+    logfile: str
+    loglevel: str
+
+    def start_logging(self):
+        logging.basicConfig(filename=self.logfile, level=LEVELS[self.loglevel])
+        logging.info("starting experiment")
+
+
+
+def create_experiment(experiment_param):
+    start_date = date(1980,1,1)
+    end_date = date(2040, 12, 31)
+    interval = timedelta(days=30)
+    population_size = 1000
+    try:
+        start_date = date(int(experiment_param['START_YEAR']),1 ,1 )
+        end_date = date(int(experiment_param['END_YEAR']),12, 31)
+        population_size = int(experiment_param['POPULATION'])
+        interval = timedelta(days = int(experiment_param['TIME_INTERVAL_DAYS']))
+    except RuntimeError as err:
```

Comment:
I'm not sure this will catch what we want. For example, `int` and `date` both raise `ValueError` if their arguments are not appropriate, so it may be better to catch that instead of `RuntimeError`.

Reply:
will replace it with ValueError